README.md

```diff
@@ -9,6 +9,63 @@ This plugin bundle provides a number of plugins to help you quickly setup a deco
 
 > ⚠️ This project is under active development. If you are a VIP customer, please let us know if you'd like to use this plugin and we can provide additional guidance. Issues and PRs are welcome. 💖
 
+## Table of contents
+- [Installation](#installation)
+  - [Install via `git subtree`](#install-via-git-subtree)
+  - [Install via ZIP file](#install-via-zip-file)
+  - [Plugin activation](#plugin-activation)
+- [Getting started](#getting-started)
+- [Configuration](#configuration)
+  - [Setting the home URL](#setting-the-home-url)
+- [Jetpack Configuration](#jetpack-configuration)
+  - [Plugin settings](#plugin-settings)
+- [Sub-plugins](#sub-plugins)
+  - [WPGraphQL](#wpgraphql)
+  - [WPGraphQL Content blocks](#wpgraphql-content-blocks)
+  - [WPGraphQL Preview](#wpgraphql-preview)
+- [Contributing](#contributing)
+
+## Installation
+
+### Install via `git subtree`
+
+The latest version of the WordPress VIP Decoupled Plugin Bundle is available in the default `trunk` branch of this repository.
+
+We recommend installing the latest plugin version [via `git subtree`][wpvip-plugin-subtrees] within your site's repository:
```

Comment:
Curious about these recommendations. Also these links don't work

Reply:
Just updated it, and simplified it down